projects/1.16-fabric/assets/charm/charm/lang/zh_cn.json

```
@@ -1 +1,143 @@
+  "block.charm.dark_horn_prismarine": "深色鹿角海晶石",
+  "block.charm.dark_oak_barrel": "深色橡木木桶",
+  "block.charm.dark_oak_crate": "深色橡木板条箱",
+  "block.charm.dark_oak_bookcase": "深色橡木书柜",
+  "block.charm.dark_oak_bookshelf": "深色橡木书架",
+  "block.charm.dark_oak_chest": "深色橡木箱子",
+  "block.charm.dark_oak_ladder": "深色橡木梯子",
+  "block.charm.dark_oak_trapped_chest": "深色橡木陷阱箱",
+  "block.charm.dark_tube_prismarine": "深色管海晶石",
+  "block.charm.ender_pearl_block": "末影珍珠方块",
+  "block.charm.fire_prismarine": "火海晶石",
+  "block.charm.fire_prismarine_bricks": "火海晶石砖",
+  "block.charm.fire_sea_lantern": "火海晶灯",
+  "block.charm.fisherman_barrel": "渔人木桶",
+  "block.charm.glowball_blob": "荧球斑点",
+  "block.charm.gold_bars": "金栏杆",
+  "block.charm.gold_chain": "金链",
+  "block.charm.gold_lantern": "金灯笼",
+  "block.charm.gold_soul_lantern": "灵魂金灯笼",
+  "block.charm.gunpowder_block": "火药方块",
+  "block.charm.horn_prismarine": "鹿角海晶石",
+  "block.charm.horn_prismarine_bricks": "鹿角海晶石砖",
+  "block.charm.horn_sea_lantern": "鹿角海晶灯",
+  "block.charm.jungle_barrel": "丛林木桶",
+  "block.charm.jungle_crate": "丛林木板条箱",
+  "block.charm.jungle_bookcase": "丛林木书柜",
+  "block.charm.jungle_bookshelf": "丛林木书架",
+  "block.charm.jungle_chest": "丛林木箱子",
+  "block.charm.jungle_ladder": "丛林木梯子",
+  "block.charm.jungle_trapped_chest": "丛林木陷阱箱",
+  "block.charm.kiln": "窑",
+  "block.charm.oak_barrel": "橡木木桶",
+  "block.charm.oak_crate": "橡木板条箱",
+  "block.charm.oak_bookcase": "橡木书柜",
+  "block.charm.oak_bookshelf": "橡木书架",
+  "block.charm.oak_chest": "橡木箱子",
+  "block.charm.oak_ladder": "橡木梯子",
+  "block.charm.oak_trapped_chest": "橡木陷阱箱",
+  "block.charm.player_pressure_plate": "玩家压力板",
+  "block.charm.redstone_lantern": "红石灯笼",
+  "block.charm.redstone_sand": "红石沙",
+  "block.charm.spruce_barrel": "云杉木桶",
+  "block.charm.spruce_bookcase": "云杉木书柜",
+  "block.charm.spruce_bookshelf": "云杉木书架",
+  "block.charm.spruce_crate": "云杉木板条箱",
+  "block.charm.spruce_chest": "云杉木箱子",
+  "block.charm.spruce_ladder": "云杉木梯子",
+  "block.charm.spruce_trapped_chest": "云杉木陷阱箱",
+  "block.charm.sugar_block": "糖块",
+  "block.charm.tube_prismarine": "管海晶石",
+  "block.charm.tube_prismarine_bricks": "管海晶石砖",
+  "block.charm.tube_sea_lantern": "管海晶灯",
+  "block.charm.warped_barrel": "诡异木桶",
+  "block.charm.warped_bookcase": "诡异木书柜",
+  "block.charm.warped_bookshelf": "诡异木书架",
+  "block.charm.warped_crate": "诡异木板条箱",
+  "block.charm.warped_chest": "诡异木箱子",
+  "block.charm.warped_ladder": "诡异木梯子",
+  "block.charm.warped_trapped_chest": "诡异木陷阱箱",
+  "block.charm.woodcutter": "切木机",
+  "container.charm.crate": "板条箱",
+  "container.charm.kiln": "窑",
+  "container.charm.portable_crafting_table": "便携式工作台",
+  "container.charm.portable_ender_chest": "便携式末影箱",
+  "container.charm.woodcutter": "切木机",
+  "enchantment.charm.acquisition": "收集",
+  "enchantment.charm.acquisition.desc": "工具自动拾取掉落物",
+  "enchantment.charm.tinted": "染色",
+  "enchantment.charm.tinted.desc": "允许在铁砧用染料改变附魔闪光的颜色",
```

```suggestion
  "enchantment.charm.tinted.desc": "允许在铁砧用染料改变附魔光效的颜色",
```
这个是有魔咒后允许改还是改后自动加上这个魔咒

---

有魔咒以后，可以在铁砧用染料改

---

那就改成这样吧，`光效`是wiki的用词

---

那就改成上面这样吧，`光效`是wiki的用词

---

已改

---

最好在`铁砧`后面加个`处`，加不加随意